script.js

```diff
@@ -1,16 +1,105 @@
 // Write your JavaScript code here!
+window.addEventListener("load", function(){
+    let listedPlanets;
+    // Set listedPlanetsResponse equal to the value returned by calling myFetch()
+    listedPlanets = "https://handlers.education.launchcode.org/static/planets.json";
```

Comment:
```suggestion
```
Instead of setting `listedPlanets` to this URL itself, you will want to use your `myFetch()` functionin `scriptHelper.js` to pass the URL inside `fetch()` to get the response from it.